### PR TITLE
LPS-163462 [PRM] Create contact table from MDF Request Detail

### PR DESCRIPTION
**Pull request by EuniceTravassos**

### CODE REVIEW

**Comment by ytuik** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+			},
+		}
+	);
+	if (response.ok) {
+		const data = await response.json();
+		const firstName = data.r_userToMDFRequests_user.givenName;
+		const infoEmail = data.r_userToMDFRequests_user.emailAddress;
+
+		fragmentElement.querySelector('#firstName').innerHTML = `${firstName}`;
+		fragmentElement.querySelector('#infoEmail').innerHTML = `${infoEmail}`;
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js:42]:
```suggestion
		fragmentElement.querySelector('#firstName').innerHTML = htmlUtil.escape(firstName);
		fragmentElement.querySelector('#infoEmail').innerHTML = htmlUtil.escape(infoEmail);
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+/* eslint-disable no-undef */
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js:11]:
```suggestion
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js:21]:
```suggestion
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	const response = await fetch(
+		`/o/c/mdfrequests/${mdfRequestId}?nestedFields=user`,
+		{
+			headers: {
+				'accept': 'application/json',
+				'x-csrf-token': Liferay.authToken,
+			},
+		}
+	);
+	if (response.ok) {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js:36]:
```suggestion
	);

	if (response.ok) {
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by EuniceTravassos** [comment]
Just starting reviewing :)

**Pull request closed without merging**